### PR TITLE
Menu Slide AnimationV2

### PR DESCRIPTION
**Pull request by AndreasInk2**
-Added slide animation to the menu

### CODE REVIEW

**Review by ibeckermayer — CHANGES_REQUESTED** [review]
This is looking pretty good, only issue is that when I click the menu button it appears to take an extra half second to start sliding, compared to clicking the X which reacts immediately. Do you notice the same thing, and if so do you know what that might be?

File: COVIDWatch.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -944,11 +944,11 @@
 			baseConfigurationReference = A236DDAE6CBC8116CEEE63D5 /* Pods-COVIDWatch iOS.debug.xcconfig */;
```
Comment (jeffctown) [inline COVIDWatch.xcodeproj/project.pbxproj]: Please revert all changes in this file so automation doesn't break

Comment (ibeckermayer) [reply COVIDWatch.xcodeproj/project.pbxproj]: pushed

**Review by ibeckermayer — CHANGES_REQUESTED** [review]
A few issues to tackle before we merge this:

1. This is out of sync with the base (`dev`) branch. You need to pull `dev` and then merge it into this branch, and fix the conflicts:
```bash
git checkout dev
git pull
git checkout menuAnimation2
git merge dev
# git will tell you that this couldn't be merged cleanly
git status # will show you which files need to be merged manually
```
See this [tutorial](https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts) for a more in depth look at merge conflicts

2. The menu currently doesn't extend through the safe area on the top (this wasn't something you changed, it was already like this in `dev`). However this looks awkward, so please go ahead and make it extend through.
Screen Shot 2020-04-21 at 3 48 41 PM

3. The darkening-overlay is unnecessarily animated. Please change it so that the overlay appears immediately, and only have the menu sliding out be animated

4. When the menu slides out, it looks like the text options appear, but don't slide, while the `launch-symbol` and the Stanford Logo images slide in (slightly delayed). Choose whichever of these two you prefer, and apply it consistently to all the menu elements.

5. The width/layout of the menu items has changed such that they're no longer symmetrical across the vertical axis:
Screen Shot 2020-04-21 at 4 05 21 PM
as compared to
Screen Shot 2020-04-21 at 4 06 25 PM

6. Some of the indentation in the code is still off. Before you commit, try to remember to autoindent your files (cmd+A followed by crtl+I)

**Pull request closed without merging**